### PR TITLE
Studio instance created email text updated.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-6799](https://openedx.atlassian.net/browse/ECOM-6799)

screen shot 2017-01-12 at 2 22 58 pm

### CODE REVIEW

**Comment by waheedahmed** [comment]
@tasawernawaz, @awais786 Please review.

**Review by tasawernawaz — APPROVED** [review]

**Comment by waheedahmed** [comment]
@bderusha Please review this.

File: course_discovery/templates/publisher/email/studio_instance_created.html [hunk 1]
```diff
@@ -2,21 +2,47 @@
 {% load i18n %}
 {% block body %}
 <!-- Message Body -->
-        <p>
-             <!--Translators: course_team_name is course team member name. .-->
-            {% blocktrans trimmed %}
-                Dear {{ course_team_name }},
+    <p>
+         <!--Translators: course_team_name is course team member name. .-->
```
Comment (bderusha) [inline course_discovery/templates/publisher/email/studio_instance_created.html]: This comment is not formatted properly to show up in the translation files.  Please see docs for proper syntax:
https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#translator-comments-in-templates

File: course_discovery/templates/publisher/email/studio_instance_created.html [hunk 2]
```diff
@@ -2,21 +2,47 @@
...
+                We expect your completed {{ link_start }}MOOC Development Checklist{{ link_end }} before your course starts. This checklist includes valuable best practices that help you to stay on track as you design and build your course. We recommend that you use the list early!
             {% endblocktrans %}
-        </p>
+        </li>
+    </ul>
 
-         <!--Translators: It's closing of mail.-->
-        <p>{% trans "Thanks," %}</p>
-        <p>{{ partner_coordinator_name }}</p>
+     <!--Translators: It's closing of mail.-->
```
Comment (bderusha) [inline course_discovery/templates/publisher/email/studio_instance_created.html]: This comment is not formatted properly to show up in the translation files.  Please see docs for proper syntax:
https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#translator-comments-in-templates

**Review by bderusha — APPROVED** [review]
Update the translation comments and merge the commits 👍

**Pull request merged**